scripts/policy/misc/stats.zeek

```diff
@@ -84,6 +88,57 @@ export {
 	global log_stats: event(rec: Info);
 }
 
+global bytes_received_cf = Telemetry::register_counter_family([
+    $prefix="zeek.net_stats",
+    $name="bytes_received",
+    $unit="1",
+    $help_text="Total number of bytes received",
+]);
+
+global packets_received_cf = Telemetry::register_counter_family([
+    $prefix="zeek.net_stats",
+    $name="packets_received",
+    $unit="1",
+    $help_text="Total number of packets received",
+]);
+
+global packets_dropped_cf = Telemetry::register_counter_family([
+    $prefix="zeek.net_stats",
+    $name="packets_dropped",
+    $unit="1",
+    $help_text="Total number of packets dropped",
+]);
+
+global link_packets_cf = Telemetry::register_counter_family([
+    $prefix="zeek.net_stats",
+    $name="link_packets",
+    $unit="1",
+    $help_text="Total number of packets on the packet source link before filtering",
+]);
+
+global packets_filtered_cf = Telemetry::register_counter_family([
+    $prefix="zeek.net_stats",
+    $name="packets_filtered",
+    $unit="1",
+    $help_text="Total number of packets filtered",
+]);
+
```

Comment:
I'm now wondering if a `gauge` `zeek_net_packet_lag_seconds` would be cool, too :-) But can do as a follow-up or when it comes up.